Automattic/woocommerce-payments
# In-person payments: attach level 3 data when capturing payments

Fixes #1962, #1944

#### Changes proposed in this Pull Request

- Changes the way merchant country is provided for level-3 data generation
- Turned out that level-3 data was already supplied when capturing, so we needed only to add the new attribute there

#### Testing instructions

- Ensure merchant account country is US
- Make a regular purchase with authorizing payments setting on
- Capture the payment from the order details page
- Check stripe dashboard logs and ensure level-3 data were part of the capture event and contains the new property (customer_reference)

-------------------
